pkg/client/client.go

```diff
@@ -416,3 +442,27 @@ func (c *Client) indexes(tableName string) ([][]string, []string, error) {
 
 	return c.Query(query, args...)
 }
+
+func getDB(driver, connString, database string) (*sqlx.DB, error) {
+	var newConnString string
+
+	if driver == drivers.MySQL {
+		newConnString = strings.Replace(connString, "/", fmt.Sprintf("/%s", database), 1)
+	} else {
+		u, err := url.Parse(connString)
+		if err != nil {
+			return nil, err
+		}
+
+		u.Path = "/" + database
+		newConnString = u.String()
+	}
+
+	db, err := sqlx.Open(driver, newConnString)
+	if err != nil {
+
+		panic(err)
```

Comment:
Why a panic?


```suggestion
		return nil, err
```

Reply:
I left it there by accident. I'll remove it,
Thanks!